apps/src/code-studio/initApp/loadApp.js

```diff
@@ -98,6 +102,22 @@ export function setupApp(appOptions) {
     },
     onAttempt: function (report) {
       if (appOptions.level.isProjectLevel && !appOptions.level.edit_blocks) {
+        // Add the frame to the drawing.
+        const frame = new Image();
+        const imageData = new Image();
+        imageData.src = `data:image/png;base64,${decodeURIComponent(report.image)}`;
```

Comment:
It feels like there's a chunk of implementation detail here that doesn't belong in initApp.js.  Could we extract a helper, maybe to [imageUtils.js](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/imageUtils.js)?